chunjun-connectors/chunjun-connector-oraclelogminer/src/main/java/com/dtstack/chunjun/connector/oraclelogminer/listener/LogParser.java

```diff
@@ -230,9 +234,26 @@ public static String parseString(String value) {
             return value.substring(15, value.length() - 2);
         }
 
+        //support nchar、nvarchar2 chinese value
+        if (value.startsWith("UNISTR('") && value.endsWith("')")) {
+            String substring = value.substring(8, value.length() - 2);
+            String replace = substring.replace("\\", "\\u");
+            return unicodeToString(replace);
+        }
+
         return value;
     }
 
+    public static String unicodeToString(String str) {
```

Comment:
Can you add a unit test for this?

Reply:
package com.dtstack.chunjun.local.test;

import com.dtstack.chunjun.connector.oraclelogminer.listener.LogParser;

/**
 * @author liuche
 * @date 2022/8/26 11:37
 */
public class LogParserTest {
    public static void main(String[] args) {
        String str="UNISTR('\\5927\\6D77')";
        str = LogParser.parseString(str);
        System.out.println(str);
    }
}